test-data/environments/asymmetric_vqgan.environment.yaml

```diff
@@ -0,0 +1,32 @@
+# https://github.com/buxiangzhiren/Asymmetric_VQGAN/blob/406c5cd2e86b3e9565795cc5cd1e3aab304b1a44/environment.yaml
+#name: ldm
+channels:
+  - pytorch
+  - defaults
+dependencies:
+  - python=3.8.5
+  - pip=20.3
+  - cudatoolkit=11.0
+  - pytorch=1.7.0
+  - torchvision=0.8.1
+  - numpy=1.19.2
+  - pip:
+      - albumentations==0.4.3
+      - easydict==1.10
+      - scikit-learn==1.2.0
+      - opencv-python==4.1.2.30
+      - pudb==2019.2
+      - imageio==2.9.0
+      - imageio-ffmpeg==0.4.2
+      - pytorch-lightning==1.4.2
+      - omegaconf==2.1.1
+      - test-tube>=0.7.5
+      - streamlit>=0.73.1
+      - einops==0.3.0
+      - torch-fidelity==0.3.0
+      - transformers==4.6.0
+      - torchmetrics==0.6
+      - academictorrents==2.3.3
+#    - -e git+https://github.com/CompVis/taming-transformers.git@master#egg=taming-transformers
```

Comment:
Would be great if we could also read these when they are formatted like:
```
- -e "taming-transformers @ git+https://github.com/CompVis/taming-transformers.git@master#egg=taming-transformers"
```

Reply:
We can :) We just parse it as a string!

Reply:
I uncommented them to showcase this